helpers/interactions/responses/sendInteractionResponse.ts

```diff
@@ -1,8 +1,7 @@
-import type { Bot } from "../../bot.ts";
-import { Embed, Message } from "../../mod.ts";
-import { DiscordMessage } from "../../types/discord.ts";
-import { AllowedMentions, FileContent, MessageComponents } from "../../types/discordeno.ts";
-import { InteractionResponseTypes } from "../../types/shared.ts";
+import type { Bot } from "../../../bot.ts";
+import { Embed } from "../../../mod.ts";
+import { AllowedMentions, FileContent, MessageComponents } from "../../../types/discordeno.ts";
+import { InteractionResponseTypes } from "../../../types/shared.ts";
 
 /**
```

Comment:
This function if it is split up, should be moved to the helpers plugin that just handles re-routing to the appropriate function